lib/takeaway.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'smsmessager'
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :menu
+
+  def initialize
+    @menu = { chicken_noodle_soup: 4.95,
+      duck_spring_rolls: 4.95,
+      crispy_duck: 8.95,
+      singapore_fried_noodles: 7.95,
+      steamed_rice: 2.95 }
+  end
+
+  def complete_order(order_total)
+    SmsMessager.send_text("Thank you! Your order total of £#{order_total} was placed and will be delivered before #{delivery_time}")
```

Comment:
Metrics/LineLength: Line is too long. [132/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)